### DISCUSSION TITLE
Обработка клика по маркеру кластера очень близких точек

### DISCUSSION THREAD

**Discussion by YaredFall** [yandex-maps-unofficial/vue-yandex-maps#375] [Q&A (2.x)]
Стоит такая задача: при нажатии на маркер отображать на экране информацию о месте (например о ПВЗ или магазине). Для отображения маркеров так же используется кластеризатор с параметром `zoom-on-cluster-click`.

Однако есть нюанс: некоторые точки расположены буквально в одном месте, с минимальной разницей в координатах. Соответственно, индивидуальные маркеры не могут быть отображены ни при каком значении зума,
они всегда выводятся в кластере.

Вопрос: как обработать клик по маркеру кластера так, чтобы определить, что дальнейший зум не возможен, и вывести на экран данные, привязанные к входящим в кластер маркерам.

Буду благодарен, если подскажете решение.

**Comment by daniluk4000** [comment]
Здравствуйте! Пытался реализовать аналогичную задачу, решения не придумал

Можете попытаться в settings передать maxZoom и настроить так -- кластер будет отключаться при указанном зуме

**Reply by daniluk4000** [reply]
Либо играться с gridSize, но если элементы НАСТОЛЬКО близко, работать будет плохо

**Reply by YaredFall** [reply]
Да, проблема что элементы буквально в одном здании, и адекватно отобразить индивидуальные маркеры проблематично.

Жаль, но спасибо за ответ! Будем думать...

**Comment by cat11box** [comment]
> maxZoom: Если зум карты превышает maxZoom, маркеры будут выведены как-есть (без кластеризации).
Есть подозрение, что эта штука вообще не работает. Несколько точек, у которых координаты отличаются на 0,00001 при любом зуме отображаются как кластер.

**Comment by cat11box** [comment]
Коллеги, подскажите, есть какая-то возможность управлять кластеризацией? Можно её как-то включать/отключать вообще?
Потому что при маленьком зуме и большом количестве поинтов без кластеризации карта тормозит. А при большом зуме кластеры только мешают.
